.changeset/famous-clocks-rule.md

```diff
@@ -0,0 +1,5 @@
+---
+'sku': patch
+---
+
+`vite`: Emit more accurate HMR metric
```

Comment:
Changeset could include more details of the value "Record metrics as observed by browser"

Reply:
https://github.com/seek-oss/sku/pull/1211/commits/8d7e7308a3468e4a9d4add5065d2dac648ffa4cc